### DISCUSSION TITLE
Granular permission [Feb 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5111] [Discord Threads]
> 🙋‍♂️ **shikishikichangchang**

**Comment by o1lab** [comment]
> 🙋‍♂️ **shikishikichangchang**  
#### Is there a way where I can control the permissions of CRUD for each table?  
#  
> 🙋‍♂️ **shikishikichangchang**  
#### Eg, some table I want my team to be able to delete the rows and some I do not want them to be able to do that  
#  
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### currently you can mark read-write permissions at project level for each user using access control (creator/editor/commenter/viewer). at table level for each role, you can control visibility though  
#  
> 🙋‍♂️ **shikishikichangchang**  
#### So there’s no table level control for CRUD? When you say visibility, do you mean visibility for CRUD for each table?  
#  
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### visibility : whether to show/hide a given table in left navigation tree for a role. for a user with role permissions set to `editor`, you can control list of tables that he/she will be able to see/edit.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1061119789931712543)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
